components/EventList.js

```diff
@@ -15,6 +16,37 @@ export default class EventList extends React.Component {
         await this.props.reload();
     }
 
+    scrollToNow = () => {
+        let currentHour = new Date();
+        currentHour.setMinutes(0, 0, 0);
+
+        const { sections } = this.props;
+        let i = 0;
+        for (i = 0; i < sections.length; i++) {
+            let sectionData = sections[i].data
+            let startDate = new Date(sectionData[0].start);
+            if (startDate >= currentHour) {
+                break;
+            }
+        }
+
+        if (i >= sections.length) {
+            return;
+        }
+
+        this.sectionListRef.scrollToLocation({
+            itemIndex: -1,
```

Comment:
What's the -1 for?

Reply:
I think there's a bug, and something like this should get the correct item and section indices 

```
const getCurrentItem(sections, currentHour) {
    let i = 0;
    let j = 0;
    for (i = 0; i < sections.length; i++) {
        let sectionData = sections[i].data
        for (j = 0; j < sectionData.length; j++) {
            let startDate = new Date(sectionData[0].start);
            if (startDate >= currentHour) {
                return [i, j];
            }
        }
    }
    return [i-1, j-1]
}
```

Reply:
Really what we're doing is wanting to jump to the current section. At least, that's how the previous app had it implemented. I guess the way this app is set up is that it is sectioned by day. The previous app was sectioned by hour. So we should either section the list by hour or implement the above version. For some reason I just assumed last night that it was by hour.